lib/models/user.dart

```diff
@@ -4,5 +4,27 @@ class User {
   final String nome;
   final String cognome;
 
-  User(this.nome, this.cognome);
+  final String codiceFiscale;
+  final DateTime dataNascita;
+  final String indirizzo;
+  final String telefono;
+  final String email;
+  final String tipoDiabete;
+  final String comorbilita;
+  final String farmaciAssunti;
+  final int periodoDiMonitoraggio;
```

Comment:
Some attributes like 'sesso' and 'numeriUtili' are missing. Furthermore isn't it better naming the class as "Patient"? Since tutors will be users too.

Reply:
Yeah, those are just models made to fill the UI for now, I'm starting to implement the api service + dto + real entities this afternoon

Reply:
I won't change backend naming, if it's called patient there, it will be called patient here